<launch>
	
	<!--arg name = "rvizconfig" default="$(find scara_kinematics)/rviz/urdf.rviz"/-->
	<!--arg name="rviz_model" default="$(find scara_kinematics)/urdf/arm_rviz.urdf"/-->
	<arg name="gazebo_model" default="$(find scara_kinematics)/urdf/arm_gazebo.urdf"/>
	<arg name="use_gui_arg" default = "true"/>

	<!--node name="forward_kinematics" pkg="scara_kinematics" type="forward_kinematics.py"/-->
	<!--node name="invkin_server" pkg="scara_kinematics" type="ivkin_server.py"/-->
	<!--node name="gazebo_kin_interface" pkg="scara_kinematics" type="gazebo_kinematics_interface.py"/-->
	<node name ="PDController" pkg="scara_kinematics" type="PDController.py"/>
	<node name ="3_move_robot_srv" pkg="scara_kinematics" type="3_move_robot_srv.py"/>
	<node name ="3_PDController" pkg="scara_kinematics" type="3_PDController.py"/>
	<node name ="CalcJointVel" pkg="scara_kinematics" type="CalcJointVel.py"/>
	<node name ="CalcJointVel_client" pkg="scara_kinematics" type="CalcJointVel_client.py"/>
	<node name ="CalcTaskVel" pkg="scara_kinematics" type="CalcTaskVel.py"/>
	<node name ="CalcTaskVel_client" pkg="scara_kinematics" type="CalcTaskVel_client.py"/>


	<include file="$(find scara_kinematics)/launch/gazebo_arm.launch">
	    <arg name="gui" value="$(arg use_gui_arg)" />
	    <arg name="model" value="$(arg gazebo_model)" />
	    <arg name="use_clock_frequency" value = "true"/>
	    <arg name="pub_clock_frequency" value = "1"/>
	  </include>

	<!--include file="$(find MYROBOT_control)/launch/arm_control.launch"/-->

</launch>

